### PR TITLE
[Java] Add support for shape list attributes

### PR DESCRIPTION
**Pull request by karllessard**
This allows passing a list of shapes as an attribute to an operation. Some operation wrappers cannot be generated without it.

### CODE REVIEW

File: tensorflow/java/src/test/java/org/tensorflow/OperationBuilderTest.java [hunk 1]
```diff
@@ -175,6 +188,27 @@ public void addControlInput() {
     }
   }
 
+  private static void testSetAttrShapeList(Shape[] shapes) {
```
Comment (asimshankar) [inline tensorflow/java/src/test/java/org/tensorflow/OperationBuilderTest.java]: Somewhat unfortunate that the tests require the use of operations that we are discouraging :). The Dataset API (`tf.data` module in Python) better serves most uses of queues.

**Review by asimshankar — APPROVED** [review]
Thanks!

**Comment by asimshankar** [comment]
Jenkins, test this please

File: tensorflow/java/src/test/java/org/tensorflow/OperationBuilderTest.java (hunk 1 above)
Comment (karllessard) [reply tensorflow/java/src/test/java/org/tensorflow/OperationBuilderTest.java]: Yeah I tried to use datasets at first (though I didn't know queues were deprecated) but didn't find an easy way for testing it, if you have an idea, I can change it later.

Comment (asimshankar) [reply tensorflow/java/src/test/java/org/tensorflow/OperationBuilderTest.java]: Not deprecated, just that there is less of a reason to use them given that the most common use case is better served by Datasets. Anyway, this is fine for now.

**Pull request merged**